server/badge.go

```diff
@@ -2,6 +2,7 @@ package server
 
 import (
 	"fmt"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Just a tiny nitpick, put that one line below with a blank line separated from the core imports.